### PR TITLE
delete article endpoint test 

### PR DESCRIPTION
**Pull request by beblicarl**
#### Pre-Submission Checklist

-   [x] Your pull request targets the `main` branch.
-   [x] Branch name is descriptive and follows the guidelines [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices).
-   [x] All tests pass . Use `git commit --amend` to amend any fixes.

#### Checklist:

-   [x] All tests are passing.
-   [x] Tested changes locally.
-   [x] Improved documentation or readme where necessary.
-   [x] Your commit messages format follows our commit message guideline [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices#commits).

#### What does this PR do?

It adds test to delete article endpoint
#### How should this be manually tested?

1. Start the test with `npm run test`
2. This should return all tests passing

#### What are the relevant issues?
We had to test for the delete article endpoint

### CODE REVIEW

**Comment by beblicarl** [comment]
@jocrah , kindly review this for me

File: src/routes/articles/delete-article.e2e.test.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+		it('should return 200', async () => {
+			const post = await fixtures.insertPost({
+				userId : user.id , 
+				type : 'article'
+			})
+			await fixtures.insertPostFlag({
+				userId : user.id , 
+				postId : post.id}) 
+			return fixtures.api()
+				.delete(`/api/v1/articles/${post.id}/flags/${user.id}`)
```
Comment (jocrah) [inline src/routes/articles/delete-article.e2e.test.js:54]: kindly check this. Not sure this is endpoint we're testing

Comment (beblicarl) [reply src/routes/articles/delete-article.e2e.test.js:54]: an oversight, thanks a lot

Comment (beblicarl) [reply src/routes/articles/delete-article.e2e.test.js]: i have pushed the changes

File: src/routes/articles/delete-article.e2e.test.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+		  }
+			return	fixtures.api()
+				.delete(`/api/v1/articles/sdsdsd`)
+				.set('Authorization', `Bearer ${accessToken}`)
+				.expect(400, expectedError)
+
+		})
+
+		it('should return 401 if request is not authenticated', async () => {
+			const post = await fixtures.insertPost({
```
Comment (jocrah) [inline src/routes/articles/delete-article.e2e.test.js:36]: the request is not authenticated and won't work anyway so we don't have to insert a post

Comment (beblicarl) [reply src/routes/articles/delete-article.e2e.test.js:36]: yes please

File: src/routes/articles/delete-article.e2e.test.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+	})
+
+	describe('Success', () => {
+
+		it('should return 200', async () => {
+			const post = await fixtures.insertPost({
+				userId : user.id , 
+				type : 'article'
+			})
+			await fixtures.insertPostFlag({
```
Comment (jocrah) [inline src/routes/articles/delete-article.e2e.test.js:50]: Kindly check this if we need it for the test

**Review by jocrah — APPROVED** [review]

**Pull request merged**